proposals/lip-0053.md

```diff
@@ -92,7 +92,7 @@ These points guarantee that the CCMs are always forwarded to the correct chains
 The [execution of cross-chain messages][lip-0049#apply] as part of the processing of a cross-chain update is done in several stages, in a way analogous to the execution of a transaction as part of the [processing of a block][lip-0055#block-processing-stages]:
 
 1. **Cross-chain message verification**: Each module can define protocol logic that verifies a CCM, possibly by accessing the state store. If an error occurs, the channel with the CCU sending chain is terminated and the CCU execution stops.
-2. **Cross-chain command verification**: The cross-chain command corresponding to the CCM `module`-`crossChainCommand` combination is verified. If an error occurs, the channel with the CCM sending chain is terminated and the CCM execution stops.
+2. **Cross-chain command verification**: The cross-chain command corresponding to the CCM `module`-`crossChainCommand` combination is verified. If an error occurs, the channel with the CCM sending chain is terminated and the CCM execution stops. Note that before verifying the cross-chain command, the parameters are decoded and validated against the schema.
```

Comment:
I think we should say this at the beginning, so that it is clear that also in this case if an error occurs the channel is terminated etc